apps/election-manager/src/components/ExportElectionBallotPackageModalButton.tsx

```diff
@@ -104,23 +104,33 @@ const ExportElectionBallotPackageModalButton: React.FC = () => {
   )
 
   // Callback to open the file dialog.
-  const openFileDialog = async () => {
+  const saveFileCallback = async (openDialog: boolean) => {
     if (state.type !== 'ArchiveBegin') {
       throw new Error(
-        `unexpected state '${state.type}' found during openFileDialog callback`
+        `unexpected state '${state.type}' found during saveFileCallback`
       )
     }
     try {
-      await state.archive.begin({
-        defaultPath: defaultFileName,
-        filters: [{ name: 'Archive Files', extensions: ['zip'] }],
-      })
+      const usbPathRaw = await getDevicePath()
+      const usbPath = usbPathRaw ? `${usbPathRaw}/` : ''
+      if (openDialog) {
+        await state.archive.beginWithDialog({
+          defaultPath: `${usbPath}ballot-export-packages/${defaultFileName}`,
```

Comment:
This and the directory path are repeated a few times. I'd consolidate and use `path.join` to build these paths rather than manual string manipulation.